packages/core-orders/db/orders/schema.js

```diff
@@ -47,7 +47,7 @@ Orders.attachSchema(
 );
 
 Migrations.add({
-  version: 20200915.2,
+  version: 20210525.1,
```

Comment:
You should create a new Migrations.add( and always keep the old one.

But anyways we should get away from the migrations when it comes to index creation, it was inflicting too many errors in the past. Do it like here: https://github.com/unchainedshop/unchained/blob/40570b5bb6a233dc0891b7fba3c64c5aae3b0ef4/packages/core-users/db/schema.js#L261

So it will try to create all indexes and if something goes wrong it just removes all indexes and then tries it again

Reply:
got it